hw/ip/otp_ctrl/data/BUILD

```diff
@@ -155,51 +172,66 @@ otp_json(
+                    A = CONST.ALERT.ESC_NONE,
+                    B = CONST.ALERT.ESC_NONE,
+                    C = CONST.ALERT.ESC_NONE,
+                    D = CONST.ALERT.ESC_NONE,
+                ),
+                # Set the classifications to kAlertClassX.
                 # See `alert_class_t` in
                 # sw/device/silicon_creator/lib/drivers/alert.h
-                "OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": ["0x94949494"] * 80,
-                # Set the classifiactions to kAlertClassX. See `alert_class_t` in
+                "OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": otp_alert_classification(
+                    alert_list = EARLGREY_ALERTS,
+                    default = alert(
+                        dev = "X",
+                        prod = "X",
+                        prod_end = "X",
+                        rma = "X",
+                    ),
+                ),
+
+                # Set the classifications to kAlertClassX. See `alert_class_t` in
                 # sw/device/silicon_creator/lib/drivers/alert.h
-                "OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": ["0x94949494"] * 16,
+                "OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": otp_alert_classification(
+                    alert_list = EARLGREY_LOC_ALERTS,
+                    default = alert(
+                        dev = "X",
+                        prod = "X",
+                        prod_end = "X",
+                        rma = "X",
+                    ),
+                ),
                 # Set the alert accumulation thresholds to 0 per class.
-                "OWNER_SW_CFG_ROM_ALERT_ACCUM_THRESH": ["0x00000000"] * 4,
+                "OWNER_SW_CFG_ROM_ALERT_ACCUM_THRESH": otp_per_class_ints(A = 0, B = 0, C = 0, D = 0),
                 # Set the alert timeout cycles to 0 per class.
-                "OWNER_SW_CFG_ROM_ALERT_TIMEOUT_CYCLES": ["0x00000000"] * 4,
+                "OWNER_SW_CFG_ROM_ALERT_TIMEOUT_CYCLES": otp_per_class_ints(A = 0, B = 0, C = 0, D = 0),
                 # Set the alert phase cycles to 0,10,10,0xFFFFFFFF for classes
                 # A and B, and to all zeros for classes C and D.
-                "OWNER_SW_CFG_ROM_ALERT_PHASE_CYCLES": [
-                    "0x0",
-                    "0xa",
-                    "0xa",
-                    "0xFFFFFFFF",
-                    "0x0",
-                    "0xa",
-                    "0xa",
-                    "0xFFFFFFFF",
-                    "0x0",
-                    "0x0",
-                    "0x0",
-                    "0x0",
-                    "0x0",
-                    "0x0",
-                    "0x0",
-                    "0x0",
-                ],
+                "OWNER_SW_CFG_ROM_ALERT_PHASE_CYCLES": otp_per_class_lists(
+                    A = [0x0, 0xa, 0xa, 0xffffffff],
+                    B = [0x0, 0xa, 0xa, 0xffffffff],
+                    C = [0x0, 0x0, 0x0, 0x0],
+                    D = [0x0, 0x0, 0x0, 0x0],
```

Comment:
Note, buildifier is going to reformat this so that each argument and list is fully expanded to one item per line.  Although I don't particularly like it, I think we could live with that reformatting.

We could always revert to string processing if we didn't like it:
```
"OWNER_SW_CFG_ROM_ALERT_PHASE_CYCLES": otp_per_class_lists(
    A = "0x0, 0xa, 0xa, 0xffffffff",
    B = "0x0, 0xa, 0xa, 0xffffffff",
    C = "0x0, 0x0, 0x0, 0x0",
    D = "0x0, 0x0, 0x0, 0x0",
)
```

Reply:
Given that we're doing string processing to the problem above, I'll also modify this macro to use strings as well.